### PR TITLE
[CON-139] Convert CaSH symbols and strings to Operators

### PR DESCRIPTION
**Pull request by dearyhud**

### CODE REVIEW

File: concourse/src/main/java/org/cinchapi/concourse/lang/OperatorSymbol.java [hunk 1]
```diff
@@ -55,13 +55,12 @@ public static OperatorSymbol create(Operator operator) {
...
      */
     public static OperatorSymbol parse(String string) {
-        for (Operator operator : Operator.values()) {
-            if(string.equalsIgnoreCase(Convert.operatorToString(operator))
-                    || string.equalsIgnoreCase(operator.name())) {
-                return new OperatorSymbol(operator);
-            }
-        }
-        throw new RuntimeException(MessageFormat.format(
+    	Operator operator = Convert.stringToOperator(string);
```
Comment (jtnelson) [inline concourse/src/main/java/org/cinchapi/concourse/lang/OperatorSymbol.java]: Add braces to your if and else blocks to maintain consistency with the rest of the code-style :)

**Comment by jtnelson** [comment]
Looks good, only need to make formatting changes to remain consistent with the rest of the code. I'd recommend installing the formatter at https://raw.githubusercontent.com/cinchapi/dev-tools/master/CinchapiFormatter.xml for all the Concourse projects.

**Comment by dearyhud** [comment]
Cool, I cloned cinchapi/dev-tools and applied the formatting to each of my local concourse projects. That way, when I hit save the code is formatted automatically. If it checks out, i'll be ready for the next issue :)

File: concourse/src/main/java/org/cinchapi/concourse/util/Convert.java [hunk 2]
```diff
@@ -31,8 +31,9 @@
 
 import javax.annotation.concurrent.Immutable;
 
-import org.cinchapi.concourse.Tag;
+import org.cinchapi.concourse.Concourse;
```
Comment (jtnelson) [inline concourse/src/main/java/org/cinchapi/concourse/util/Convert.java]: is this import used for anything?

**Comment by dearyhud** [comment]
Yea, I think the formatter alphabetized all the imports. None of them were removed.

**Comment by jtnelson** [comment]
Eclipse doesn't give a warning about an unused import?

**Comment by dearyhud** [comment]
Nope. No errors. I can send a screenshot of that helps

**Comment by jtnelson** [comment]
yeah my bad i read the diff wrong :+1:

**Pull request merged**